situp-core/src/integrationTest/java/com/amazon/situp/integration/EndToEndIntegrationTest.java

```diff
@@ -70,41 +79,97 @@ public void testPipelineEndToEnd() throws IOException, InterruptedException {
 
         //Verify data in elasticsearch sink
         final List<Map<String, Object>> expectedDocuments = getExpectedDocuments(exportTraceServiceRequest1, exportTraceServiceRequest2);
+        final List<Map<String, Object>> expectedTargets = IntStream.range(0, serviceNames.size()).mapToObj(i -> {
```

Comment:
Wait, why are we doing this? This seems like a deterministic set of expected targets and edges for a random set of data?